e2e/terraform/shared/config/provision-client.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# installs and configures the desired build of Nomad as a server
+set -eu
```

Comment:
could you expand these to the long form versions of the flags for easier readability?

Reply:
Done!